packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -94,8 +94,10 @@
     background-color: var(--color-surface);
     box-shadow: inset 0 0 0 1px var(--gray-a5);
 
-    &:where(:not([data-state='checked']):hover) {
-      box-shadow: inset 0 0 0 1px var(--gray-a7);
+    @media (hover: hover) {
+      &:where(:not([data-state='checked']):hover) {
+        box-shadow: inset 0 0 0 1px var(--gray-a7);
+      }
```

Comment:
`postcss-hover-media-feature` in WorkDS attempted to add `@media (hover: hover)`, but with complex selectors like this it tends to get into some endless loop crashing the node servers